lib/net/ssh/verifiers/always.rb

```diff
@@ -34,6 +34,10 @@ def verify(arguments)
           found
         end
 
+        def verify_signature(arguments)
+          return arguments[:key].ssh_do_verify(arguments[:signature], arguments[:hash])
```

Comment:
I think we can use keyword arguments as net-ssh is ruby2.2+ i think

Reply:
OK, will update